packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -119,7 +119,7 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({ isTablet, patientUuid }
                 id="visitDateTimePicker"
                 labelText={t('date', 'Date')}
                 placeholder="dd/mm/yyyy"
-                style={{ width: '100%' }}
+                style={{ width: '80%' }}
```

Comment:
I honestly don't what is the best alternative for this.
cc @ibacher

Reply:
what happens if we still reduce the width, could the whole date be appearing on minimizing the screen @jnsereko

Reply:
@HerbertYiga, the more we reduce the date, the less the date values (`19/02/2021`) are seen.  Instead, they get masked by the date icon on window minimisation.

Reply:
@jnsereko its best to  wrap this into two rows such that date and time inputs are on different rows rather than try to fit them on one row when the width is too small to support such styling.

Reply:
@jnsereko its best to  wrap this into two rows such that date and time inputs are on different rows rather than try to fit them on one row when the width is too small to support such styling.